samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md

```diff
@@ -30,18 +31,43 @@ developers to test and debug their bots on localhost or running remotely through
 - Launch the Bot Framework Emulator.
 - File -> Open bot and navigate to `botbuilder-samples/samples/csharp_dotnetcore/05.multi-turn-prompt` folder.
 - Select `BotConfiguration.bot` file.
+# Use Azure Blob Storage or Azure CosmosDB
+
+##### In Visual Studio

```

Comment:
Same comment - I don't think this section is needed.

Reply:
We removed the `Azure Blob Storage` section from the Readme.